### DISCUSSION TITLE
ADetailer: img2img inpainting detected. adetailer disabled.

### DISCUSSION THREAD

**Discussion by iiiiik06** [Bing-su/adetailer#454] [Q&A]
the prompt（ADetailer: img2img inpainting detected. adetailer disabled.）,How to solve this problem?

**Comment by sgmgong2** [comment]
same problem after update:<

**Reply by iiiiik06** [reply]
So I switched back to the previous version.

**Reply by jcvijr** [reply]
Same here. Which command you use for switching to previous version?
Also, I don't know if this is a bug or a fix. It it is a fix, it should have an option to disable in img2img tab...

[EDIT] as a workaround, while we don't have the correct answer:

At file scripts/!adetailer.py , change line 638 from True to False , right above the message "[-] ADetailer: img2img inpainting detected. adetailer disabled."

It worked again.

**Reply by iiiiik06** [reply]
The author wrote in the log about this disabling issue, and I addressed it by overwriting with the compressed file from the previous version.

**Reply by iiiiik06** [reply]
I attempted to modify it to "False," but it doesn't seem to be working.

**Reply by jcvijr** [reply]
Instead of hardcoding, I used these commands in my jupyter notebook setup, as a temporary fix:

```
#!After Detailer
!git clone https://github.com/Bing-su/adetailer /content/stable-diffusion-webui/extensions/adetailer

#temporary fix of this: https://github.com/Bing-su/adetailer/issues/458
%cd /content/stable-diffusion-webui/extensions/adetailer

!git reset --hard 98d075d73994cfc4d7c2590702618bd145683b55 #back to functional adetailer commit

!git clean -df #(cleans any untracked files/folders)

%cd /content/stable-diffusion-webui

!git clone https://huggingface.co/Bingsu/adetailer /content/stable-diffusion-webui/models/adetailer
```

---

If you have a local install, just use the git reset and git clean commands inside adetailer folder.

**Comment by Skys2023** [comment]
Same issue, how can you revert back?

**Reply by iiiiik06** [reply]
I tried various methods, but none of them worked, so I decided to revert to the previous version.

**Reply by Skys2023** [reply]
Thanks iiiik06, to do this, do I download the adetailer-v23.11.1.zip file, (and ignore the tar.gz file) , unzip it, go to stable-difusion-webui\extensions folder and remove the previous adetailer, and past in v23.11.1 ? 

I noticed inside the stable-difusion-webui\extensions folder, there are a couple subdirectories called _pycache_, will these be recreated?

**Reply by iiiiik06** [reply]
I'm not sure about this,I haven't looked into it.

**Reply by cherchesss** [reply]
same problem. what shall I do?

**Comment by LucianoDaluz** [comment]
Working perfectly after the 21/02/24 update.

**Comment by LucianoDaluz** [comment]
Having some other issues now,how can I back to the previous version?
